app/gamify/model.js

```diff
@@ -0,0 +1,12 @@
+//  application: {type: mongoose.Schema.Types.ObjectId, ref: 'Application'},
```

Comment:
Not exactly sure why but I think you need `'use strict'` here to avoid the error